public/pages/DetectorResults/components/DetectorState/DetectorUnknownState.tsx

```diff
@@ -47,11 +47,7 @@ export const DetectorUnknownState = (props: DetectorUnknownStateProps) => {
         >
           View detector configuration
         </EuiButton>,
-        <EuiButton
-          onClick={props.onStartDetector}
-          iconType={'play'}
-          style={{ width: '250px' }}
-        >
+        <EuiButton onClick={props.onStartDetector} style={{ width: '250px' }}>
```

Comment:
I see we removed the icon in file `DetectorStopped.tsx` as well, but the width changed to '200px' from '250px'. Should we change the width here?

Reply:
Good catch, will update.